recipes/simhash-py/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "simhash-py" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "c83e20cfac873ba9067c564135c293db635cbd7372f126e9bcfb0fabf9fd0234" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/27/42/fc4203f6881e98a744765f7911d783bbd67d5c85706e7e66d1c06e6c6534/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - toolchain
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - simhash
+
+about:
+  home: http://github.com/seomoz/simhash-py/
+  license: MIT
+  license_family: MIT
```

Comment:
Please add `# license_file: LICENSE` with a note that it will be included in the next release and thus should be uncommented then.